### ISSUE TITLE
Alternative to Plotly

### ISSUE THREAD

**Issue by ruuseljo** [ruuseljo/schoolprojectCO2#13]
Plotly requires a =n internet connection

**Comment by rjkrussell** [comment]
is this referring to the need for the raspberry pi to reach out to the internet to render?

**Comment by ruuseljo** [comment]
Yes, I unplugged the Ethernet cable to the router. Pi was still connected
to router but no Html was generated

**Comment by rjkrussell** [comment]
sounds like u could be jumping to conclusions.  it doesn't seem right that plotly has a dependency upon the internet.  do u have a specific error message that proves your point?

**Comment by ruuseljo** [comment]
Bob, I think you are correct slthough when disconnect ethernet to router I get a Html error. Have run in debug but did not figure out the issue. Note to self: Do not jump to conclusions and use debugger in Pycharm!

Refer to https://plotly.com/python/is-plotly-free/ re need for internet connection.
